Packs/Whois/ReleaseNotes/1_1_0.md

```diff
@@ -1,7 +1,7 @@
 
-### Playbooks
-- __TIM - Process Domain registrant With Whois__
-- __TIM - Process Domain Age With Whois__
-- __TIM - Process Domains With Whois__
-    - Added three new playbooks.
+#### Playbooks
+##### __TIM - Process Domain registrant With Whois__
+##### __TIM - Process Domain Age With Whois__
+##### __TIM - Process Domains With Whois__
+- Added three new playbooks.
```

Comment:
should this line be here?

Reply:
just updated the format, didn't want to change anything else.